test/test_prototype_transforms.py

```diff
@@ -187,66 +184,6 @@ def test__get_params(self, mocker):
         assert params["needs_pad"]
         assert any(pad > 0 for pad in params["padding"])
 
-    @pytest.mark.parametrize("needs", list(itertools.product((False, True), repeat=2)))
-    def test__transform(self, mocker, needs):
```

Comment:
We remove this test, because it never checked correctness, but rather if the specific dispatchers were being called. Due to our new architecture, they are never called and thus this test fails.

The same argument applies to the removed tests in `test/test_transforms_v2.py`.